<launch> 

	<!-- Start all the nodes that are to be under test -->
	<include file="$(find babs_lidar_wobbler)/launch/start_wobbler.launch" /> 

	<!-- Start all the nodes that support simulation of the system -->
	<include file="$(find babs_nav)/launch/init_sim.launch" /> 

	<!-- Start node that will test the wobbler nodes -->
	<node pkg="babs_wobbler_tests" type="wobbler_integration_test" name="wobbler_integration_test" args="" output="screen" > 

	<!-- Test params to test the test --> 

	<param name="hokuyo_scan_name" value="/front_wobbler/scan" />
	<param name="transformed_scan_cloud_name" value="/front_wobbler/scan_cloud" />
	<param name="stitched_cloud_name" value="/front_wobbler/point_cloud" />
	<param name="start_testing" value="1.00" />

	</node>

	<!-- ACTIVATE THIS NODE FOR IF NON PHYSICAL -->
	<!-- Start point cloud rosbag play to take place of input data streams -->
	<node pkg="rosbag" type="play" name="rosbag_babs_scan_record" args="/home/mordoc/wobbler_scans/input.bag" respawn="true" >\
		<remap from="/scan" to="/front_wobbler/scan"/>
		<remap from="/dynamixel_motor1_ang" to="/front_wobbler/angle"/>
	</node>

	<!-- TEMPLATE
	<node pkg=" "  type=" " name=" " output="screen" > 
	</node> -->

</launch>
